openapi_tester/utils.py

```diff
@@ -58,3 +59,17 @@ def lazy_combinations(options_list: Sequence[dict[str, Any]]) -> Iterator[dict]:
     for i in range(2, len(options_list) + 1):
         for combination in combinations(options_list, i):
             yield merge_objects(combination)
+
+
+def is_path_an_url(path: str) -> bool:
```

Comment:
Think we could use the django url validator directly: https://github.com/django/django/blob/stable/1.3.x/django/core/validators.py#L45

Reply:
Nice, I hadn't considered it. Changed it to use django's validator.